<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CpaymentAccountDlg</class>
 <widget class="QDialog" name="CpaymentAccountDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="m_accountName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Account Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_paymentAccountName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Payment Account Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_natureOFExpense">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nature of Expense</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_okBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
